File: Assignment 8/Activity 1.js
function main() {
    // this program uses a loop to generate a list of multiplication expressions for a given value
    var number;
    var var_input;
    var increment;
    
    number = getValue("number");
    var_input = getValue("input");
    increment = 1;
    whileLoop(number, var_input, increment);
}

function getValue(name) {
    var value;
    
    window.alert(" Enter " + name + " value: ");
    value = window.prompt('Enter a value for value');
    
    return value;
}

function whileLoop(number, var_input, increment) {
    window.alert(" While loop multiplication from " + number + " to " + var_input + " by " + increment + " : ");
    var product;
    
    product = number;
    while (increment <= var_input) {
        product = number * increment;
        window.alert(number.ToString() + " * " + increment + " = " + product);
        increment = increment + 1;
    }
}

File: Assignment 8/Activity 1.py
# this program uses a loop to generate a list of multiplication expressions for a given value

def getValue(name):
    print(" Enter " + name + " value: ")
    value = int(input())    
    return value


def displayExpression(number, increment, product):
    print(str(number) + " * " + str(increment) + " = " + str(product))


def processExpressions(number, factor):
    print(" While loop multiplication from " + str(number) + " to " + str(factor) + " by 1 :")
    increment = 1
    product = number
    for increment in range(increment, factor + 1, 1):
        product = number * increment
        displayExpression(number, increment, product)


def main():
    number = getValue("value")
    factor = getValue("expressions")
    processExpressions(number, factor)


main()
